packages/ui-date-time-input/src/DateTimeInput/__tests__/DateTimeInput.test.tsx

```diff
@@ -685,4 +685,42 @@ describe('<DateTimeInput />', async () => {
     })
     expect(await dateTimeInput.find(':contains(5/1/2017, 1:30 PM)')).to.exist()
   })
+
```

Comment:
Could you please add tests for the function versions too (both the message and the`disabledDates` prop)? It would be nice to test that these dates actually get disabled in the dropdown too.

Reply:
done